client/jetpack-cloud/sections/partner-portal/credit-card-fields/style.scss

```diff
@@ -0,0 +1,134 @@
+.credit-card-fields {
+	display: none;
+	margin-top: 1rem;
+	margin-bottom: 1rem;
+
+	&--is-loaded, &--placeholder {
```

Comment:
I don't know if it's strictly taken documented in any "Calypso code standards" but in the rest of the file each new selector is defined on a new line 😄